### PR TITLE
Added examples

### PR DESCRIPTION
**Pull request by patmann03**
Examples were added.  Ran into the case where some people didn't realize they could do this with the Monoprice 6 zone amp.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by patmann03** [comment]
Anyone able to help with the Test / Lint markdown.  It's asking for a language check, however I don't see how to add that to the page.  I've looked at other pages and don't see it marked either?

File: source/_integrations/monoprice.markdown [hunk 1]
```diff
@@ -55,3 +55,66 @@ Restore a previously taken snapshot of one or more speakers. If `entity_id` is `
 | Service data attribute | Optional | Description |
 | ---------------------- | -------- | ----------- |
 | `entity_id` | yes | String or list of strings that point at `entity_id`s of zones.
+
+
+## Examples
+
+### Script
+
+```
```
Comment (klaasnicolaas) [inline source/_integrations/monoprice.markdown]:
```suggestion
```yaml
```

File: source/_integrations/monoprice.markdown [hunk 2]
```diff
@@ -55,3 +55,66 @@ Restore a previously taken snapshot of one or more speakers. If `entity_id` is `
...
+        - media_player.kitchen
+        - media_player.upstairs_master
+        - media_player.great_room
+        - media_player.garage
+mode: single
+icon: mdi:speaker-message
+```
+
+### Automations
+```
```
Comment (klaasnicolaas) [inline source/_integrations/monoprice.markdown]:
```suggestion

```yaml
```

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

**Comment by patmann03** [comment]
@klaasnicolaas Thank you!

File: source/_integrations/monoprice.markdown [hunk 3]
```diff
@@ -55,3 +55,67 @@ Restore a previously taken snapshot of one or more speakers. If `entity_id` is `
...
+## Examples
+
+### Script
+
+```yaml
+# Take Snapshot of current state, turn on zones, select source.
+alias: Notify - Media Announment
+sequence:
+  - service: monoprice.snapshot
+    data: {}
```
Comment (frenck) [inline source/_integrations/monoprice.markdown]:
```suggestion
```

File: source/_integrations/monoprice.markdown [hunk 4]
```diff
@@ -55,3 +55,67 @@ Restore a previously taken snapshot of one or more speakers. If `entity_id` is `
...
 | `entity_id` | yes | String or list of strings that point at `entity_id`s of zones.
+
+
+## Examples
+
+### Script
+
+```yaml
+# Take Snapshot of current state, turn on zones, select source.
+alias: Notify - Media Announment
```
Comment (frenck) [inline source/_integrations/monoprice.markdown]:
```suggestion
alias: "Notify - Media Announcement"
```

File: source/_integrations/monoprice.markdown [hunk 5]
```diff
@@ -55,3 +55,67 @@ Restore a previously taken snapshot of one or more speakers. If `entity_id` is `
...
+    data: {}
+    target:
+      entity_id:
+        - media_player.basement
+        - media_player.garage
+        - media_player.great_room
+        - media_player.kitchen
+        - media_player.upstairs_master
+  - service: media_player.turn_on
+    data: {}
```
Comment (frenck) [inline source/_integrations/monoprice.markdown]:
```suggestion
```

File: source/_integrations/monoprice.markdown [hunk 6]
```diff
@@ -55,3 +55,67 @@ Restore a previously taken snapshot of one or more speakers. If `entity_id` is `
...
+    target:
+      entity_id:
+        - media_player.kitchen
+        - media_player.great_room
+        - media_player.garage
+        - media_player.upstairs_master
+        - media_player.basement
+  - service: media_player.select_source
+    data:
+      source: Server
```
Comment (frenck) [inline source/_integrations/monoprice.markdown]:
```suggestion
      source: "Server"
```

File: source/_integrations/monoprice.markdown [hunk 7]
```diff
@@ -55,3 +55,67 @@ Restore a previously taken snapshot of one or more speakers. If `entity_id` is `
...
+    target:
+      entity_id:
+        - media_player.basement
+        - media_player.kitchen
+        - media_player.upstairs_master
+        - media_player.great_room
+        - media_player.garage
+mode: single
+icon: mdi:speaker-message
```
Comment (frenck) [inline source/_integrations/monoprice.markdown]:
```suggestion
```

File: source/_integrations/monoprice.markdown [hunk 8]
```diff
@@ -55,3 +55,67 @@ Restore a previously taken snapshot of one or more speakers. If `entity_id` is `
...
+```
+
+### Automations
+
+```yaml
+alias: Notify - Doorbell
```
Comment (frenck) [inline source/_integrations/monoprice.markdown]: I'm not really sure why a separate script is used in this case; it makes the more complicated than it needs to be.

I would suggest making it single automation instead.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/monoprice.markdown (hunk 3 above)
Comment (patmann03) [reply source/_integrations/monoprice.markdown]: Per your other suggestion.  I removed the script and made the requested changes.

File: source/_integrations/monoprice.markdown (hunk 8 above)
Comment (patmann03) [reply source/_integrations/monoprice.markdown]: Done.  I used a script in mine so I could reuse it for different announcements / automations.  I've simplified it for the example per your suggestion.

File: source/_integrations/monoprice.markdown [hunk 9]
```diff
@@ -55,3 +55,61 @@ Restore a previously taken snapshot of one or more speakers. If `entity_id` is `
...
+    target:
+      entity_id:
+        - media_player.basement
+        - media_player.kitchen
+        - media_player.upstairs_master
+        - media_player.great_room
+        - media_player.garage
+  - service: switch.turn_on
+    target:
+      entity_id: switch.doorbell_announce
```
Comment (frenck) [inline source/_integrations/monoprice.markdown:105]: What is this?

**Comment by patmann03** [comment]
Since the monoprice is just an amp it doesn't actually recognize media. You
have to use an external source directly plugged into it to play. This is a
command line switch that plays an MP3.

**Comment by frenck** [comment]
Right, but that isn't clear/visible from this all. Which is kind of defeating the point of the whole example?

**Comment by patmann03** [comment]
I could add a comment to help out or some blurb about how it's just an amp
so you need an external media source to play music or notification. And
explain the example.


Just seen a couple people confused on reddit so thought I'd put something
together on the documentation to help.

**Comment by frenck** [comment]
Well the goal of the example provided is, according to the description: "When Doorbell is pressed play media over speakers"

However, the example added doesn't do that.

**Comment by frenck** [comment]
Going to friendly decline this PR for now, as it has been running stale. The last comment hasn't been addressed since November and in its current state, the example doesn't do what it says it should be doing.

Feel free to re-open the PR when ready to work on it again.

../Frenck

**Pull request closed without merging**